docs/pages/blog/mui-x-v8.md

```diff
@@ -0,0 +1,358 @@
+
+### Charts Composition
+
+We've restructured our charts composition by dividing responsibilities between two dedicated components: `<ChartDataProvider />` and `<ChartsSurface />`. The `<ChartDataProvider />` now focuses solely on managing and processing your chart data, while `<ChartsSurface />` is dedicated to rendering the visual aspects. This clear separation enhances customization, improves performance, and offers a more flexible integration with your application.
+
+Learn more in our [Charts Composition Documentation](/x/react-charts/composition/).
+
+## Enhanced License Management and Telemetry
+
+Managing your license has never been easier. Our redesigned [account page](https://mui.com/store/account/) on the store allows you to view your license keys, update number of seats, renew, upgrade and more, with minimal effort.
+
+<figure>
+  <img width="606" height="300" alt="Radar charts examples" src="/static/blog/mui-x-v8/accounts-page.png" loading="lazy">
+  <figcaption>Accounts Page</figcaption>
+</figure>
+
+In addition, we’ve integrated telemetry into our system to continuously enhance your experience with our products. Using this same infrastructure, you'll soon be able to manage your license without changing your key—thanks to server-side license validation.
+
+Telemetry is released as an opt-in feature. We recommend enabling it by adding the following code to your project, close to your license validation:
+
+```js
+import { muiXTelemetrySettings } from '@mui/x-license';
+
+muiXTelemetrySettings.enableTelemetry();
+```
+
+You can find more details in our [telemetry guide](/x/guides/telemetry/).
+
+## Priority Support – A New Support Service
+
+A few months ago, we launched Priority Support with a few customers, and now we’re excited to invite more users to take advantage of this service.
+
+Priority Support offers fast, dedicated assistance with guaranteed response times, ensuring that technical issues on both MUI X and MUI Core are addressed swiftly and efficiently.
+
+For full details on the benefits and service levels of Priority Support, please see our [Support SLA](https://mui.com/legal/technical-support-sla/).
+
+If you’re interested in upgrading to Priority Support, don’t hesitate to [contact our sales team](mailto:sales@mui.com) for more information or purchase it directly through your account page.
+
+## Support for Material UI
+
+When planning this major release, our goal was to support the last two major versions of Material UI. However, we faced a tough choice: continue supporting older versions (v5 and v6) or fully embrace the innovations of **Material UI v7**. With native ESM support, improved package management, and seamless integration with modern bundlers like Vite and webpack, v7 represents the future of Material UI. Supporting both wasn’t feasible, so we chose to move forward.
```

Comment:
```suggestion
When planning this major release, our goal was to support the last two major versions of Material UI. However, we faced a tough choice: continue supporting older versions (v5 and v6) or fully embrace the innovations of **Material UI v7**. With native ESM support, improved package management, and seamless integration with modern bundlers like Vite and webpack, v7 represents the future of Material UI.
```

This statement is a bit misleading – it suggests it's not possible to use Material UI v5/v6.